karma.conf.js

```diff
@@ -33,11 +31,6 @@ module.exports = function(config) {
       devtool: 'inline-source-map',
       node: {
         crypto: false
-      },
-      resolve: {
-        alias: {
-          crypto$: path.resolve(__dirname, 'no-crypto.js')
```

Comment:
Can the no-crypto.js file be removed too?

Reply:
I'm not sure. I'll try removing it and see if the tests still pass, if so, I'll remove it.

Reply:
Oh, sorry, misread that, it's already been removed, I'll remove the file.